scripts/components/api-changes-validator/api_changes_validator.test.ts

```diff
@@ -52,7 +52,7 @@ void describe('Api changes validator', { concurrency: true }, () => {
           'utf-8'
         )
       ).trim();
-      await execa('tsc', ['--build'], { cwd: latestPackagePath });
+      await execa('npx', ['tsc', '--build'], { cwd: latestPackagePath });
```

Comment:
this is to match https://github.com/aws-amplify/amplify-backend/blob/ad99a960e70ebb8cdf823d87a2dd1dad99345379/scripts/components/api-changes-validator/api_changes_validator.ts#L55 .

it was unhappy locally otherwise after I got rid of global tsc